src/main/java/org/fogbowcloud/saps/engine/core/catalog/jdbc/exceptions/JDBCCatalogException.java

```diff
@@ -0,0 +1,10 @@
+package org.fogbowcloud.saps.engine.core.catalog.jdbc.exceptions;
+
+public class JDBCCatalogException extends RuntimeException {
+
+    private static final long serialVersionUID = -2520888793776997437L;
+
+    public JDBCCatalogException(String msg) {
+        super(msg);
+    }
+}
```

Comment:
I believe that this exception must inherit from the `CatalogException`. `JDBCCatalogException` is a specific type of exception for Catalog implementations.

Reply:
Done